tests/test_data_migrations.py

```diff
@@ -642,6 +644,33 @@ def migrator(response: ResponseInfo):
         )
         assert resp.status_code == 200
 
+    def test__fastapi_response_migration__response_is_streaming_response_and_there_is_a_migration(
+        self,
+        create_versioned_clients: CreateVersionedClients,
+        test_path: Literal["/test"],
+        latest_module: ModuleType,
+        router: VersionedAPIRouter,
+    ):
+        @router.post(test_path, response_model=latest_module.AnyResponseSchema)
+        async def post_endpoint(request: Request):
+            return StreamingResponse(StringIO(), status_code=200)
```

Comment:
Let's add some bytes there to check them in tests

Reply:
Added